### DISCUSSION TITLE
Need Two Children in template

### DISCUSSION THREAD

**Discussion by anjanesh** [alpinejs/alpine#3544] [1. Help]
I need to do something like this :
```
<tbody data-accordion="table-column">
    <template x-for="index in $store.global_data.length">
        <tr :id="'table-column-header-' + index" :data-accordion-target="'#table-column-body-' + index" aria-expanded="false" :aria-controls="'table-column-body-' + index">
        </tr>
        <tr :id="'table-column-body-' + index" :aria-labelledby="'table-column-header-' + index">
        </tr>
    </template>
</tbody>
```

Is it possible for a template to have 2 children (tr tags) ?

**Comment by ekwoka** [comment]
You should be able to do this with an element that has `display: contents` wrapping them.

But templates should only have one root element.

Alpine might be able to support it, but it does make all the loop logic more difficult.
